doc/metrics_catalog.md

```diff
@@ -1,3 +1,7 @@
+# Metrics catalog
+
+This document catalogs the Prometheus metrics instrumentation for xmptd.
```

Comment:
Are these Prometheus metrics?

Reply:
good question. Our goal is to be openmetrics, but we are actually not OM compliant. Currently we are Prometheus. 

For clarity, OM is prometheus with a bunch of extra restrictions.

I will check whether we can switch some flags to make us OM. It is the goal.

Reply:
https://github.com/xmtp/xmtpd/pull/809

- the docs can now safely say it is openmetrics :)